src/display/api.js

```diff
@@ -213,6 +213,8 @@ const DefaultStandardFontDataFactory =
  *   when creating canvases. The default value is {new DOMCanvasFactory()}.
  * @property {Object} [filterFactory] - A factory instance that will be used
  *   to create SVG filters when rendering some images on the main canvas.
+ * @property {boolean} [enableHardwareAcceleration] - Enables hardware
```

Comment:
*Name bike-shedding:* Since HWA should be a well known acronym, how about using a slightly shorter name through this patch? 
```suggestion
 * @property {boolean} [enableHWA] - Enables hardware
```